tests/keras/layers/test_convolutional.py

```diff
@@ -270,6 +270,22 @@ def test_globalpooling_2d():
 
 
 @keras_test
+def test_globalpooling_3d():
+    layer_test(pooling.GlobalMaxPooling3D,
+               kwargs={'dim_ordering': 'th'},
+               input_shape=(3, 4, 11, 12, 10))
+    layer_test(pooling.GlobalMaxPooling3D,
+               kwargs={'dim_ordering': 'tf'},
+               input_shape=(3, 4, 11, 12, 10))
+    layer_test(pooling.GlobalAveragePooling3D,
+               kwargs={'dim_ordering': 'th'},
+               input_shape=(3, 4, 11, 12, 10))
+    layer_test(pooling.GlobalAveragePooling3D,
+               kwargs={'dim_ordering': 'tf'},
+               input_shape=(3, 4, 11, 12, 10))
```

Comment:
Use much smaller dimensions in order to speed up the tests (3-4, single channel).